### PR TITLE
Remove disabled encrypted-media/Google tests

### PR DESCRIPTION
**Pull request by foolip**
These files all still exist here:
https://cs.chromium.org/chromium/src/third_party/WebKit/LayoutTests/media/encrypted-media/

With the single exception of encrypted-media-utils.js, they all have
some differences, i.e. they have diverged since this copy was made.

Since they are disabled, these tests don't show up in wpt.fyi:
http://wpt.fyi/encrypted-media

Additionally, they are excluded from import back into Chromium:
https://cs.chromium.org/chromium/src/third_party/WebKit/LayoutTests/W3CImportExpectations

In summary, they aren't being run by anyone and aren't providing value.
It would be more effective to upstream these anew and resolve any issues
as part of that process.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/7582)

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-10-04 17:24:21
Finished: 2017-10-04 17:36:04


<h2>Failing Jobs</h2>
<ul>

<li>safari:10.0</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/19158)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/283293579)

**Review by plehegar — APPROVED** [review]

**Comment by foolip** [comment]
@ddorwin, can you do a check after the fact? I'll simply revert if this does not make sense to you.

**Comment by ddorwin** [comment]
LGTM
